src/views/payment-sheet-views/card-view.js

```diff
@@ -356,26 +427,31 @@ CardView.prototype.showFieldError = function (field, errorMessage) {
   fieldError = this.fieldErrors[field];
   fieldError.textContent = errorMessage;
 
-  this.hostedFieldsInstance.setAttribute({
-    field: field,
-    attribute: 'aria-invalid',
-    value: true
-  });
+  if (!(input && isNormalFieldElement(input))) {
+    this.hostedFieldsInstance.setAttribute({
+      field: field,
+      attribute: 'aria-invalid',
+      value: true
+    });
+  }
 };
 
 CardView.prototype.hideFieldError = function (field) {
   var fieldGroup = this.getElementById(camelCaseToKebabCase(field) + '-field-group');
+  var input = fieldGroup.querySelector('input');
 
   if (!this.fieldErrors.hasOwnProperty(field)) {
     this.fieldErrors[field] = this.getElementById(camelCaseToKebabCase(field) + '-field-error');
   }
 
   classlist.remove(fieldGroup, 'braintree-form__field-group--has-error');
 
-  this.hostedFieldsInstance.removeAttribute({
-    field: field,
-    attribute: 'aria-invalid'
-  });
+  if (!(input && isNormalFieldElement(input))) {
+    this.hostedFieldsInstance.removeAttribute({
+      field: field,
+      attribute: 'aria-invalid'
+    });
+  }
```

Comment:
We probably want to set and remove aria-invalid on normal inputs as well.

Reply:
👍